.github/workflows/publish.yml

```diff
@@ -101,8 +110,6 @@ jobs:
           EMAIL: bot@getsentry.com
           GITHUB_API_TOKEN: ${{ steps.token.outputs.token }}
           COCOAPODS_TRUNK_TOKEN: ${{ secrets.COCOAPODS_TRUNK_TOKEN }}
-          CRAFT_GCS_TARGET_CREDS_JSON: ${{ secrets.CRAFT_GCS_TARGET_CREDS_JSON }}
-          CRAFT_GCS_STORE_CREDS_JSON: ${{ secrets.CRAFT_GCS_STORE_CREDS_JSON }}
```

Comment:
Am I right to assume that these still need some values from the new auth step you added?

Reply:
These won't be needed since the credentials will be derived dynamically with https://cloud.google.com/docs/authentication/application-default-credentials

Reply:
Don't know about GCS itself but Craft will 100% throw an exception: https://github.com/getsentry/craft/blob/92fbd36486d57551a0bbeacd72624f44c4016989/src/utils/gcsApi.ts#L73-L88

Reply:
For history, support of Google ADC in Craft was implemented in https://github.com/getsentry/craft/pull/600